plugins/pepkong/pepkong-0.0-2.rockspec

```diff
@@ -0,0 +1,28 @@
+
+package = "pepkong"
+version = "0.0-2"
+
+local pluginName = "pepkong"
+
+source = {
+  url = "https://github.com/dojot/kong.git"
+}
+description = {
+  summary = "A Kong plugin, that extract roles from a JWT token and make a request for a XACML Policy Decision Point (PDP)",
```

Comment:
**XACML**: Would it be correct to keep this acronym in this description? I am not sure, but it seems to me that we are not implementing this specification and its standardization, just taking advantage of _terminologies_ such as PEP, PDP, etc ...